### DISCUSSION TITLE
메일서버 (송신, 발신) 설정

### DISCUSSION THREAD

**Discussion by ZZ3n** [100-hours-a-week/discussions#60] [개발] [answered]
# 문제 상황
1. 메일 발신, 수신 기능이 필요합니다.
2. 현재 메일 발신은 gmail smtp를 통해 나가고 있습니다.
   - 카터 계정으로 발신 중, ( {카터 계정}@gmail.com )
3. 현재 메일 수신은 따로 하고 있지 않은 상태입니다.

# 해결 하고 싶은 문제 
메일 발신, 수신을 개인 도메인 계정으로 하고 싶습니다. ( noreply@qqqq.world, help@qqqq.world 처럼 )

1. 어떤 방법으로 하는 게 좋을까요?
2. 구글 비즈니스로 하면 되는 것 같던데 역시 돈을 쓰는게 제일 좋을까요?

**Comment by ej31** [comment, marked as answer]
## 답변

> 메일 발신, 수신을 개인 도메인 계정으로 하고 싶습니다. ( [noreply@qqqq.world](mailto:noreply@qqqq.world), [help@qqqq.world](mailto:help@qqqq.world) 처럼 )

- AWS SES 이용하시는게 제일 심플하게 해결할 수 있는 상황이에요. 말씀하신 설정이 모두 가능합니다.
- SES 에서 자격증명 얻은 뒤에 DNS 업체쪽 설정에서 Record 설정 착착해주면 됩니다.
- [링크](https://techblog.gccompany.co.kr/%EC%95%88%EB%85%95%ED%95%98%EC%84%B8%EC%9A%94-%EC%98%AC%ED%95%B4-1%EC%9B%94%EC%97%90-%EC%97%AC%EA%B8%B0%EC%96%B4%EB%95%8C-sre%ED%8C%80%EC%97%90-%EC%83%88%EB%A1%AD%EA%B2%8C-%ED%95%A9%EB%A5%98%ED%95%98%EA%B2%8C-%EB%90%9C-%EC%97%98%EB%A6%AC%EC%9E%85%EB%8B%88%EB%8B%A4-6334db671cab) 참고하세요!
---
<img width="900" alt="image" src="https://github.com/user-attachments/assets/022cc381-b085-4d58-a32c-49630078e5ad">
